Regression/WarpX-tests.ini

```diff
@@ -62,6 +62,7 @@ buildDir = .
 inputFile = Examples/Tests/PML/inputs2d
 runtime_params = warpx.do_dynamic_scheduling=0 warpx.maxwell_fdtd_solver=yee
 dim = 2
+addToCompileString =
```

Comment:
I added `addToCompileString = ` to each tests, in order to easily modify the `WarpX-tests.ini` for GPU, with an automated script that adds `USE_GPU=TRUE` to `addToCompileString`.